pkg/shipper/shipper.go

```diff
@@ -124,6 +140,18 @@ func SendBatch(files []string, config configs.Configuration) {
 	}
 
 	log.Print("Conexion closed")
+
+	if len(processedFiles) > 0 {
+		registry.StoreFilesIntoRegistry(processedFiles, *memoryRegistry, registryFileLocation)
+	}
+}
+
+// We need to access to the files of a nested map[string]interface{} data, for this we should use a type of reflection
+// to access to every key of the nested map
+func appendFilesInSliceOfBatch(files *[]string, batch []interface{}) {
+	for i := 0; i < len(batch); i += 1 {
+		*files = append(*files, batch[i].(map[string]interface{})["log"].(map[string]interface{})["file"].(map[string]interface{})["path"].(string))
```

Comment:
Someone will probably ask if there's another way to do it. No. I didn't find any other way to access to nested `map[string]interface{}` fields.